src/Devices.php

```diff
@@ -190,7 +190,7 @@ public function onFocus($id, array $data)
      *
      * @return array
      */
-    public function csvExport(array $extraFields = [])
+    public function csvExport(array $extraFields = [], $segmentName = null, $lastActiveSince = null)
```

Comment:
Adding new arguments would be a BC break :/

Reply:
If both fields are option (which they are), I believe this won't be BC break. It will be a minor release for sure, but not BC break -- in my opinion

Reply:
Probably ok.

Reply:
thanks both. Yes, I made the parameters optional to avoid BC breakage. I'll update with your suggestions and resubmit.